browser/about_flags.cc

```diff
@@ -281,6 +281,12 @@ constexpr char kBraveRewardsVerboseLoggingDescription[] =
     "and access tokens depending on your activity. Please do not share it "
     "unless asked to by Brave staff.";
 
+constexpr char kBraveRewardsAllowUnsupportedWalletProvidersName[] =
+    "Show unsupported Brave Rewards custodial options";
```

Comment:
Nit: maybe instead of "custodial options", "custodial wallet options"? (And similar for the description text.)

Reply:
I'm okay with both. I'll leave that up to @Miyayes, as both the name and description are coming from him. 🙂

Reply:
>Always show Brave Rewards custodial connection options